docs/user-guide/how_to_configure_scheduler.md

```diff
@@ -0,0 +1,114 @@
+* `tiers` divides the plugins into several categories. All the functions defined in the plugins will be registered when
+a session is open and called when actions are executed. 
+* In some scenarios, users may configure different plugins which registers the same functions. It will depend on the business
+requirement to decide how to combine these functions. That's why `tier` is required.
+
+## Actions
+* `Action` implements the main logic of scheduling. 
+* Volcano allow users to make self-defined actions.
+* Volcano provides 7 built-in actions until April 2022. The details are as follows.
+
+| ID  | Name     | Required | Description                                                                                                                                                                                                                                                           |
+|-----|----------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1   | enqueue  | Y        | Judge whether the idle resource in the cluster can satisfy the basic demand of a workload. If yes, set the podgroup of the workload to be `inqueue`, otherwise keep the podgroup `pending`. Notice that the default value for parameter `overcommit-factor` is `1.2`. |
+| 2   | allocate | Y        | Try to allocate resource to workloads whose corresponding podgroup status is `inqueue`.                                                                                                                                                                               |
+| 3   | backfill | N        | Try to allocate resource to workloads whose pods are `BestEffort`.                                                                                                                                                                                                    |
+| 4   | preempt  | N        | Recognise workloads with high priority. Try to evict pods with low priority and allocate the resource to them.                                                                                                                                                        |
+| 5   | reclaim  | N        | Pick out queues whose resources have been borrowed by other queues and reclaim them back.                                                                                                                                                                             |
+| 6   | elect    | N        | Select a workload satisfying some conditions. It is designed to work with resource reservation for target workload. Will deprecated at future releases.                                                                                                               |
```

Comment:
We will not add description for remove elect and reserve actions any more in new doc. It is depreciated.

Reply:
Yes, so I add the comment "Will deprecated at future releases". When they are deprecated from the project, I'll update it.